### DISCUSSION TITLE
Removing media from congregation sync server

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#1623] [Q&A] [answered]
Good afternoon, I have a question about server synchronization (I currently use koofr server). See the situation below:

1. I add an image on the server at home

2. If it was a wrong image, I delete it and send another one, however, if someone has already synchronized it before deleting it from the server, the image will be downloaded logically, however, the question is:
- If the image is being deleted from the server, I close the M3 that had downloaded the wrong image and synchronizing (downloading) the media again, will the wrong image remain on the PC that had downloaded before updating to the correct image?

Note: This scenario currently occurs, I sent an image to the server, the media was downloaded on KH's PC, when deleting the image on the home PC, and closing the M3, opening and synchronizing, the image remains on the PC, even deleting it from cloud. Is this a BUG or is it supposed to work anyway?
Detail: On the PC where the image was inserted and deleted from the server, it does not download again, it is only downloaded in the media of the other PC.

**Comment by mtdvlpr** [comment, marked as answer]
@e-Feitosa, this is intended behaviour. There is no way to distinguish a locally added file from a file that was removed from the congregation sync server.
